update hosted supported regions

chomipi88 opened the pull request:
https://github.com/astronomer/astro/issues/9426

jwitz commented:
@chomipi88 Was this change made on Tuesday? I'm going to add a release note for it

chomipi88 commented:
@jwitz yesterday we updated the regions as part of product packaging updates. no impact to any customers as we don't have any hosted clusters in the region we dropped support for.

jwitz commented:
Got it, I'll still write a note for the ones that we added.
